content/en/logs/guide/log-collection-troubleshooting-guide.md

```diff
@@ -26,20 +26,27 @@ Changes in the configuration of the `datadog-agent` won't be taken into account
 
 ## Outbound traffic on port 10516 is blocked
 
-The Datadog Agent sends its logs to Datadog over tcp via port 10516. If that connection is not available, logs fail to be sent and an error is recorded in the `agent.log` file to that effect.
+The Datadog Agent sends its logs to Datadog over TCP using port 10516. If that connection is not available, logs fail to be sent and an error is recorded in the `agent.log` file to that effect.
 
-Test manually your connection by running a telnet or openssl command like so (port 10514 would work too, but is less secure):
+You can manually test your connection using OpenSSL, GnuTLS, or another SSL/TLS client. For OpenSSL, run the following command:
 
-* `openssl s_client -connect intake.logs.datadoghq.com:10516`
-* `telnet intake.logs.datadoghq.com 10514`
+```shell
+openssl s_client -connect intake.logs.datadoghq.com:10516
+```
+
+For GnuTLS, run the following command:
+
+```shell
+gnutls-cli intake.logs.datadoghq.com:10516
+```
 
 And then by sending a log like the following:
 
 ```text
 <API_KEY> this is a test message
 ```
 
-- If opening the port 10514 or 10516 is not an option, it is possible to configure the Datadog Agent to send logs through HTTPS by adding the following in `datadog.yaml`:
```

Comment:
@trazfr could you just confirm that it makes sense to remove this from the doc as well?

Reply:
As you wish.

It makes sense as we no longer ask new customers to use it. There are only legacy implementations which use 10514.
On the other hand, we could keep it customers using the legacy non-TLS port, even if it is discouraged.